string step
bool interruptable
dict_object[] object
bool move_arm
bool move_base
string current_location
string destination_location